flutter/lib/common/widgets/dialog.dart

```diff
@@ -1861,3 +1861,104 @@ void enter2FaDialog(
         onCancel: cancel);
   });
 }
+
+void showWindowsSessionsDialog(
+    String type,
+    String title,
+    String text,
+    OverlayDialogManager dialogManager,
+    SessionID sessionId,
+    String peerId,
+    String sessions) {
+  dialogManager.dismissAll();
+  dialogManager.show((setState, close, context) {
+    onConnect() {
+      bind.sessionReconnect(sessionId: sessionId, forceRelay: false);
```

Comment:
Making it a parameter does not work because there is one more reconnection happening after the dialog's sessionReconnect on this line of connection.rs from the server:
 ```rust
 self.send_close_reason_no_retry("Restarting...").await;
 ```
I needed to add this to let the server process restart (with the specified user id). Otherwise, the connection was getting crashed.
After this change (of sessionReconnect's parameter), when there's another attempt to reconnect (after selecting, triggered by server), the usid will be None again, and the dialog will be shown in a loop.
Please let me know if I misunderstood.

Reply:
both `self.send_close_reason_no_retry("Restarting...").await;` and `bind.sessionReconnect(sessionId: sessionId, forceRelay: false);` make the client reconnect,  why are these two needed?

Reply:
Yes, it is correct.

Reply:
Saving the chosen session id to LoginHandler, it won't change during reconnection.